woocommerce/woocommerce
# Enhancement: Tax settings conflict warning

### All Submissions:






### Changes proposed in this Pull Request:



Added a warning banner to the user when the tax settings are mismatched and in conflict.

Closes #35497 .






### How to test the changes in this Pull Request:



1. Set up woocommerce and enable tax settings in general woocommerce settings
2. Go to tax settings tab
3. If any of the 3 tax display settings (Prices entered with tax , Display prices in the shop, Display prices during cart and checkout) differ, there should be a warning banner.
4. Clicking on "Use recommended settings" should change all the settings to be the same as the one set for "Prices entered with tax "


![image](https://user-images.githubusercontent.com/27843274/207850455-a906a21c-2b28-4b48-81fc-4c59260c5a17.png)



### Other information:




### FOR PR REVIEWER ONLY:
